tests/unit/test_database.py

```diff
@@ -72,6 +72,7 @@ def tests_write_prices():
         f"postgresql+psycopg://postgres:postgres@localhost:5432/mainnet"
     )
     db = Database(engine, "mainnet")
+    # list contains duplicate entry in order to test how this is handled
```

Comment:
I would make this a separate test `test_write_prices_duplicates`. Otherwise we might get stuck with complicated tests as in `solver-rewards` where we have one huge test case with lots of settlements. It then becomes difficult to add new cases and a failed test does not show why it failed.